picard/oauth.py

```diff
@@ -129,7 +131,7 @@ def is_logged_in(self):
         return self.is_authorized() and bool(self.username)
 
     def revoke_tokens(self):
-        # TODO actually revoke the tokens on MB (I think it's not implementented there)
+        # TODO actually revoke the tokens on MB (I think it's not implemented there)
```

Comment:
Token Revocation is implemented in MB since some time. See: https://musicbrainz.org/doc/Development/OAuth2#:~:text=Bearer%22%2C%0A%20%20%22refresh_token%22%3A%22GjSCBBjp4fnbE0AKo3uFu9qq9K2fFm4u%22%0A%7D-,Revoking%20a%20token,-If%20a%20refresh

Reply:
Thanks, I'll get rid of that comment then

Reply:
PICARD-2886